explainshell/web/static/js/es.js

```diff
@@ -669,10 +673,17 @@ function drawgrouplines(commandselector, options) {
         //    link.circle = {x: rrmid, y: startytop-5-unknownlinelength-3, r: 8};
         //}
         //else {
-            link.paths.push(new espath().addpoint(rr.left, 0).addpoint(rrright, 5));
-            link.paths.push(new espath().addpoint(rrright, 5).addpoint(rrright, 0));
+            link.paths.push(new espath()
+              .addpoint(rr.left, 0)
+              .addpoint(rr.left, 5)
+              .addpoint(rrright, 5)
+              .addpoint(rrright, 0)
+            );
             var rrmid = d3.round(rr.left + rr.width / 2);
-            link.lines.push({x1: rrmid, y1: 6, x2: rrmid, y2: 5+unknownlinelength+command_offsett_to_canvas});
```

Comment:
Nothing else than using the the same pattern. This is only for me to easily understand.

Revert it?